ldap/servers/plugins/rootdn_access/rootdn_access.c

```diff
@@ -513,6 +514,15 @@ rootdn_check_access(Slapi_PBlock *pb)
             return -1;
         }
     }
+
+    /*
+     *  Remaining checks are only relevant for AF_INET/AF_INET6 connections
+     */
+    slapi_pblock_get(pb, SLAPI_CONN_SERVERNETTYPE, &server_net_type);
```

Comment:
Actaully we don't need to add a whoel new request type. The data we need is already available if we ask for `SLAPI_CONN_SERVERNETADDR`, we have only to check the address family with `PR_NetAddrFamily` here.